store/tikv/region_cache.go

```diff
@@ -381,10 +435,10 @@ func (c *RegionCache) GetTiFlashRPCContext(bo *Backoffer, id RegionVerID) (*RPCC
 
 	// sIdx is for load balance of TiFlash store.
 	sIdx := int(atomic.AddInt32(&regionStore.workTiFlashIdx, 1))
```

Comment:
Is it possible that the `workTiFlashIdx` increased to the length of the array, and is used to access the slice, then panic with slice out of bound error?

Reply:
workTiFlashIdx will continue increasing, but it always mod TiFlash store count

> accessIdx := (sIdx + i) % regionStore.accessStoreNum(TiFlashOnly)